code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -506,10 +506,19 @@
 	flags_inv = HIDEHAIR|HIDEEARS
 
 /obj/item/clothing/suit/hooded/wintercoat/centcom
-	name = "centcom winter coat"
+	name = "centcom combat winter coat"
+	desc = "An armored heavy jacket with very robust armor, for Nanotrasen officers use only."
 	icon_state = "coatcentcom"
 	item_state = "coatcentcom"
-	armor = list("melee" = 40, "bullet" = 45, "laser" = 45, "energy" = 35, "bomb" = 40, "bio" = 25, "rad" = 25, "fire" = 35, "acid" = 50)
+	armor = list("melee" = 80, "bullet" = 80, "laser" = 50, "energy" = 50, "bomb" = 100, "bio" = 100, "rad" = 100, "fire" = 90, "acid" = 90)
```

Comment:
Are you sure this is admin only? Its better than basically all the station armor at no slowdown.